docs/ca/dev/preparation-for-going-live.md

```diff
@@ -23,191 +23,108 @@ We've divided the preparation into approximate timeframes, and you can adjust th
+| APP-15 | Application                    | Implement publish and sync guidelines                        | Implement applicable [Publish and Sync stability best practices](/docs/ca/dev/best-practices/best-practises-jenkins-stability.html).                                                                                                                               
+| APP-16 | Application                    | Implement general security guidelines                        | Apply [security guidelines](/docs/dg/dev/guidelines/security-guidelines.html).                                                                                                                                                                                                                                                                       |
+| APP-17 | Application                    | Implement password security guidelines                       | Make sure that you don't have any plain-text passwords, private keys, or API secrets in config files or Git repositories.                                                                                                                                                                                                                                    
+| APP-18 | Application                    | Implement credentials security guidelines                    | Minimize the use of personal credentials and choose work-specific accounts based on each environment: production, staging, testing. We highly recommend employing Centralized Credential Management to securely store and manage these credentials.                                                                                                         
+| APP-19 | Application                    | Implement secrets rotations                                  | Secrets, like API tokens, should be rotated regularly. Outline and test rotation strategies to prevent issues during live operation.                                                                                                                                                                                            
+| APP-20 | Application                    | Implement internal update security guidelines                | Install all the [security updates](/docs/about/all/releases/product-and-code-releases.html) from all Spryker packages.                                                                                                                                                                                                                                       
```

Comment:
How are security guidelines related to installing the actual packages?

Reply:
You are right. Can you please add here - https://docs.spryker.com/docs/about/all/releases/product-and-code-releases.html ? Maybe with a comment that we need to take "Security releases"

Reply:
Not sure what you mean, that link is already there